Auth0/Credentials.swift

```diff
@@ -28,24 +28,24 @@ import Foundation
 @objc(A0Credentials)
 public class Credentials: NSObject, JSONObjectPayload {
 
-    public let accessToken: String
-    public let tokenType: String
+    public let accessToken: String?
+    public let tokenType: String?
     public let idToken: String?
     public let refreshToken: String?
+    public var hasToken: Bool {
+        if self.idToken != nil || ( self.accessToken != nil && self.tokenType != nil) { return true }
+        return false
+    }
 
-    required public init(accessToken: String, tokenType: String, idToken: String? = nil, refreshToken: String? = nil) {
+    required public init(accessToken: String? = nil, tokenType: String? = nil, idToken: String? = nil, refreshToken: String? = nil) {
```

Comment:
This is sort of a breaking change so I'd need to figure out a way to communicate this, not a big one but still a breaking one